package.json

```diff
@@ -34,6 +37,7 @@
     "babel-loader": "^7.1.0",
     "babel-preset-env": "^1.5.2",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
It allowed me to use the spread operator for passing in props to component functions. I wasn't able to use it before.